runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1827,6 +1836,44 @@ gcParseCommandLineAndInitializeWithValues(J9JavaVM *vm, IDATA *memoryParameters)
 	return JNI_EINVAL;
 }
 
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+bool
+gcParseRestoreArguments(J9JavaVM *vm)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vm);
+	J9VMInitArgs* restoreArgs = vm->checkpointState.restoreArgsList;
+	IDATA index = -1;
+	IDATA result = 0;
+
+	PORT_ACCESS_FROM_JAVAVM(vm);
+
+	/* TODO: Can this be made common with gcParseSovereignArguments parsing? */
```

Comment:
I'd prefer you do something about it even in this change. There are probably a few ways to do it. For example, you can remove this option parsing from gcParseSovereignArguments and call this method at the end (or start?) of gcParseSovereignArguments. Of course, this method should have an additional parameter if it's startup or restore so it knows where to get arg list (and probably rename it somewhat). 

But think about other ways, too.

Reply:
Changed function name to `gcParseReconfigurableArguments`, now called by `gcParseSovereignArguments`